tmt/steps/execute/internal.py

```diff
@@ -443,13 +439,17 @@ def _save_process(
             logger=logger
             )
 
-        if not invocation.hard_reboot_requested:
+        if not invocation.is_guest_alive:
```

Comment:
```suggestion
        if invocation.is_guest_alive:
```

This one should not be negated, right?

Reply:
Fixed in b3faad7c79334881fedd6a3a35f0323f698e080d.